doc/reference/common-weaknesses/slot-deadline.rst

```diff
@@ -0,0 +1,15 @@
+Slot Deadline Attack
```

Comment:
This is quite confused. Are we talking about:
- DoS attacks on the on-chain parts of contracts with limited throughputs
- The specific case where a DoS attack is used to have unexpected effects due to the presence of a deadline